packages/e2e-tests/pages/patients/PatientDetailsPage/modals/ChangeDietModal.ts

```diff
@@ -0,0 +1,41 @@
+import { Locator, Page } from '@playwright/test';
+
+export class ChangeDietModal {
+  readonly page: Page;
+  readonly modalTitle!: Locator;
+  readonly dietMultiSelectInput!: Locator;
+  readonly cancelButton!: Locator;
+  readonly confirmButton!: Locator;
+  readonly formGrid!: Locator;
+
+
+  constructor(page: Page) {
+    this.page = page;
+
+    const testIds = {
+      confirmButton: 'confirmbutton-tok1',
+      cancelButton: 'outlinedbutton-95wy',
+      formGrid: 'formgrid-r4hj',
+    } as const;
+
+    for (const [key, testId] of Object.entries(testIds)) {
+      (this as any)[key] = page.getByTestId(testId);
+    }
+    this.modalTitle = page.getByRole('dialog').getByTestId('modaltitle-ojhf');
+    this.dietMultiSelectInput = page.getByTestId('formgrid-r4hj').getByTestId('multiselectinput-vf2i').nth(1);
```

Comment:
I think you should be able to use test id `localisedfield-wlag` in ChangeDietForm instead of needing to use the 2 test Ids and the nth(1) but it doesn't matter too much